src/components/chat/comment-textbox.test.tsx

```diff
@@ -0,0 +1,79 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Provider } from "mobx-react";
+import React from "react";
+import { act } from "react-dom/test-utils";
+import { ENavTab } from "../../models/view/nav-tabs";
+import { CommentTextBox } from "./comment-textbox";
+
+describe("Comment Textbox", () => {
+
+  const stores = { ui: { activeNavTab: ENavTab.kMyWork } };
```

Comment:
Should be able to remove the `stores` and all the `Provider` stuff now that it's being passed in as a prop.